base/bucket_gocb.go

```diff
@@ -147,6 +152,72 @@ func (bucket CouchbaseBucketGoCB) GetBucketCredentials() (username, password str
 	return bucket.spec.BucketName, password
 }
 
+func (bucket CouchbaseBucketGoCB) GetMetadataPurgeInterval() (int, error) {
```

Comment:
Ha yeah I definitely didn't want to build this in Sync Gateway unless we had to.  Reached out to Brett and Matt to confirm that it's not in the SDK, and Brett replied that there isn't any current SDK support.   I don't think there are too many other use cases (that would merit inclusion in the SDK), so didn't file a feature request for it.